NEWS.md

```diff
@@ -1,13 +1,32 @@
-# IxPopDyMod 0.2.0 
+# IxPopDyMod 0.3.0
```

Comment:
@dallenmidd the new release notes in this file are the main thing to review.

Reply:
This all looks great to me!